openapi/schemas/processes-job-management/statusInfo.yaml

```diff
@@ -0,0 +1,38 @@
+type: object
+required:
+   - id
+   - status
+   - type
+properties:
+   id:
+      type: string
+   processID:
+      type: string
```

Comment:
The requirements need to be revised. They used the alternate name `jobID` when referring to the `GET /jobs` responses. 

Similarly, `process` was mentioned during the meeting. 
I'm not sure if `processID` remains relevant however, because `process` would be the URI, not just the `processID` from `GET /processes/{processID}`.